exporter/googlecloudexporter/config_test.go

```diff
@@ -32,10 +32,10 @@ import (
 // setPdataFeatureGateForTest changes the pdata feature gate during a test.
 // usage: defer SetPdataFeatureGateForTest(true)()
 func setPdataFeatureGateForTest(enabled bool) func() {
-	originalValue := featuregate.IsEnabled(pdataExporterFeatureGate)
-	featuregate.Apply(map[string]bool{pdataExporterFeatureGate: enabled})
+	originalValue := featuregate.GetRegistry().IsEnabled(pdataExporterFeatureGate)
```

Comment:
I understand that the featuregate-related changes are forced, but the commit message does not match it. Perhaps rewrite the commit message to make it more relevant.
The actual change that matches the commit message is drowned in the featuregate changes.

Reply:
@tigrannajaryan I'll add a note to the merge commit if that works for you, i'd rather not wait for another build if you're ok with this.

Reply:
see the merge commit message: https://github.com/open-telemetry/opentelemetry-collector-contrib/commit/b9915f749b7ec03fabcff3200bc659e3fe9f549d

Reply:
Thanks!